src/components/landing/listing.js

```diff
@@ -3,22 +3,22 @@ import Image from "next/image";
 import styles from "./listing.module.css";
 
 function Listing({ data }) {
-  const avatarWidth = 50
-  const avatarHeight = 50
+  const avatarWidth = 200
+  const avatarHeight = 200
   return (
     <>
-      <article className={styles.listing}>
+      <article className="bg-primary-lite rounded-lg drop-shadow-md flex justify-content-center font-sans">
         <div className={styles.listing__image__container}>
-          <Image
+          <Image className="ml-8 mt-6 rounded-full"
             src="/image-placeholder.png"
             width={avatarWidth} height={avatarHeight}
             alt={`${data.community_name} avatar`}  />
         </div>
-        <div className={styles.listing__content}>
-          <h2 className={styles.listing__content__name}>{data.community_name}</h2>
-          <p className={styles.listing__content__manager}> {data.community_manager} </p>
-          <p className={styles.listing__content__description}> {data.community_description}</p>
-          <Link href={`/${encodeURIComponent(data.communitd_id)}`}  className={styles.listing__view__more}>view more
+        <div className=" ml-12 mb-8 mr-8 mt-8 rounded-lg">
```

Comment:
@Domicaren1 Did you consult the STYLE GUIDE in styling these tiles?

Reply:
yes ma